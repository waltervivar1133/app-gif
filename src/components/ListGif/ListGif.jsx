import React from 'react'
import PropTypes from 'prop-types'

export const ListGif = ({url,title}) => {

  
  return (
    <>
      <div className="card  animate__animated animate__backInLeft">
          <img src={url} alt={title}/>
          <div>{title}</div>
      </div>
    </>
  )
}

ListGif.propTypes = {
  title: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired,
}