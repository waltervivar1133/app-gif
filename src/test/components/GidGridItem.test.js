import React from 'react';
import { shallow } from "enzyme"
import { ListGif } from "../../components/ListGif/ListGif"

describe('Prueba en GridItem' , () => {

  const props = {
    title : 'un titulo',
     url : 'https://localhost/data.png'
  }

  test('debe mostrar el componente correctamente', () => {

    const wrapper = shallow( <ListGif {...props}/> )
    expect( wrapper).toMatchSnapshot();
  })

})