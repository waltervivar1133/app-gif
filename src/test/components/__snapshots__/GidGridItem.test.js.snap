// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en GridItem debe mostrar el componente correctamente 1`] = `
<Fragment>
  <div
    className="card  animate__animated animate__backInLeft"
  >
    <img
      alt="un titulo"
      src="https://localhost/data.png"
    />
    <div>
      un titulo
    </div>
  </div>
</Fragment>
`;
